### DISCUSSION TITLE
Same code but different result in reliable channel [TestTransfer_Connect_Recv_SmallBuffer2]

### DISCUSSION THREAD

**Discussion by Allgot** [ANLAB-KAIST/KENSv3#84] [Project Q&A 2. Transport Layer (TCP)]
Hello.

We are having some troubles to write codes for PA3.

We wrote the same code and ran but it resulted in the success for my computer but in the failure for my teammate's computer.

The test case is TestTransfer_Connect_Recv_SmallBuffer2. (Please check the below figures.)

Thank you.

안녕하세요 조교님.

다름이 아니라,PA3 코드를 작성하는 중에 알 수 없는 문제가 발생하여 여쭙습니다.

같은 코드를 돌렸을 때 Packet loss나 corruption이 발생하지 않아 랜덤성이 없는 reliable channel에서도 결과가 다르게 나오는 상황입니다. 

테스트 케이스는 TestTransfer_Connect_Recv_SmallBuffer2이며, 저희가 작성한 코드가 아닌 시스템 코드에서 문제가 발생한 것은 아닌가 하여 여쭤보게 되었습니다. (보다 자세한 상황 설명을 위해 아래 사진을 첨부하였습니다.)

감사합니다.

![image](https://user-images.githubusercontent.com/96973603/167359783-2cf910ae-592e-4068-b146-57472418141d.png)
<error message>

![image](https://user-images.githubusercontent.com/96973603/167360095-1eff1b9c-dc9f-416e-8f04-d3995c84af8c.png)
<teammate's wireshark result>

![image](https://user-images.githubusercontent.com/96973603/167360707-f74b7715-1812-49f9-b774-62641cbe960e.png)
<my wireshark result>

**Comment by Pusnow** [comment]
Please, send your source codes and detailed environment info (OS, compiler, etc.) to TA (cs341.ta.pj@kaist.ac.kr).
